blivet/osinstall.py

```diff
@@ -1094,6 +1099,653 @@ def get(self, key, default=None):
         return self.mappings.get(key, default)
 
 
+class InstallerStorageConfig(Blivet):
```

Comment:
I agree. I think I may have suggested the current name, but when I read the patches I found it confusing.